src/DynamoCoreWpf/Controls/StartPage.xaml.cs

```diff
@@ -367,29 +374,107 @@ private void RefreshBackupFileList(IEnumerable<string> filePaths)
             IEnumerable<string> filePaths)
```

Comment:
was this function called more than once in Dynamo lifetime, if yes, we may want to cache the deserialized Json and only update with the latest recent file's Json object

Reply:
It only runs once when Dynamo starts (it won't update the recent files after you open/close a new file, for example, that's the original behavior).

Reply:
Ok, thanks so I think this is not going to cause performance issue for now